bhyve-api/src/ioctls.rs

```diff
@@ -87,4 +87,5 @@ pub const VM_GET_CPUS: i32 = VMM_IOC_BASE | 0x1c;
 pub const VM_SUSPEND_CPU: i32 = VMM_IOC_BASE | 0x1d;
 pub const VM_RESUME_CPU: i32 = VMM_IOC_BASE | 0x1e;
 
+pub const VM_TRACK_DIRTY_PAGES: i32 = VMM_IOC_BASE | 0x20;
```

Comment:
nit: Why isn't this grouped with the entries above (`0x1e`)?

Reply:
Because semantically it's not really related to those things (tracking dirty pages feels qualitatively different than e.g. NMI injection), and the constants in this file seem to be grouped semantically. It's also not grouped that way in the kernel. But I'm happy to change it, if you'd prefer?

Reply:
I would prefer that.

The `DEVMEM_GETOFFSET` stuff is separate because it's mainly to paper over how FreeBSD and illumos bhyve handle the direct mapping of memory objects.  The rest of the ioctls relate to running to the machine, so I would keep them grouped as such.

Reply:
Good to go; line deleted.